File: problems/l/leftbeehind/leftbeehind.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    while True:
        sweet, sour = [int(val) for val in input().split()]
        if sweet == 0 and sour == 0:
            break;

        if sweet == sour:
            print("Undecided.")
        elif sweet + sour == 13:
            print("Never speak again.")
        elif sweet > sour:
            print("To the convention.")
        else:
            print("Left beehind.")


if __name__ == '__main__':
    main()

File: problems/l/leftbeehind/leftbeehind.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <bits/stdc++.h>

using namespace std;

int main() {
    int sweet, sour;

    while (true) {
        cin >> sweet >> sour;

        if (sweet == 0 && sour == 0) {
            break;
        }
        else if (sweet == sour) {
            cout << "Undecided." << endl;
        }
        else if (sweet + sour == 13) {
            cout << "Never speak again." << endl;
        }
        else if (sweet > sour) {
            cout << "To the convention." << endl;
        }
        else {
            cout << "Left beehind." << endl;
        }
    }
}